packages/flame_riverpod/README.md

```diff
@@ -0,0 +1,59 @@
+Helpers for using Riverpod in conjunction with Flame - to share state from
+the game into other parts of your application, or from other parts of your
+application into your game.
```

Comment:
It would be better to start with a brief description of what `riverpod` is (not all readers will be familiar with it), and then explain how it would fit into the Flame universe.

AFAIU, the purpose of riverpod is to maintain user-defined state, and force the widgets that depend on that state to automatically rebuild when the state changes. It is not clear from the description how this interoperates with Flame which (1) has components instead of widgets, (2) components don't need to be rebuilt.